### ISSUE TITLE
Fix case-sensitive conditions for permitted attribute 

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#553]
```js
<link rel="preload" imageSrcSet={url} as="image" imageSizes="100vw" />
```

Warn `imageSrcSet` and `imageSizes` on the `invalid-attr` rule.

Because:

Attribute names don't resolve as case-insensitive.

https://github.com/markuplint/markuplint/blob/d2ee395d917cb69b91dedb0262b1c12ddc81fb58/packages/%40markuplint/html-spec/src/spec.link.json#L77-L88

And

There are not those IDL attributes in:
https://github.com/markuplint/markuplint/blob/d2ee395d917cb69b91dedb0262b1c12ddc81fb58/packages/%40markuplint/parser-utils/src/idl-attributes.ts#L116-L134

Related: #525

**Comment by YusukeHirao** [comment]
Resolve in https://github.com/markuplint/markuplint/releases/tag/v2.11.1
